cmd/release/internal/release.go

```diff
@@ -56,6 +57,7 @@ func NewRelease(inputBranch, inputEnvironment string) (*Release, error) {
 }
 
 // NewReleaseWithDefaultEnvironment returns complete Release based on the provided inputBranch
+// TODO: rename to indicate it's both environments and check that numbers are the same.
```

Comment:
Does this need to be renamed for this PR?

Reply:
I ended up dealing with this in another PR that's been merged